### ISSUE TITLE
slide left causing an odd "jump" after animation on android

### ISSUE THREAD

**Issue by deebloo** [Telerik-Verified-Plugins/NativePageTransitions#59]
right works like a charm but left is giving me an weird issue

**Comment by EddyVerbruggen** [comment]
Can you share a simple demo app reproducing this problem please?

**Comment by deebloo** [comment]
Yes I will try to get the sample up so I can share. Also I was looking at the android device monitor and it seems that with each transition it is adding more and more to the layout and never removing it.

**Comment by occurr** [comment]
I have the same issue FYI, and also am able to sent a built demo app your way :)

**Comment by deebloo** [comment]
+1

**Comment by deebloo** [comment]
apologies for no response

**Comment by EddyVerbruggen** [comment]
The first one to send me a simple sample app reproducing this issue gets 3 karma points for free!
